### DISCUSSION TITLE
Can I use the area information on the map?

### DISCUSSION THREAD

**Discussion by tjahnn** [fleaflet/flutter_map#1241] [Q&A] [answered]
Is it possible to get area information of a specific building or place on the map?

![image](https://user-images.githubusercontent.com/29188211/169342714-e1d0a997-b643-41b7-85f1-a3460ee0b2aa.png)

Thank you!

**Comment by ibrierley** [comment]
Not unless you somehow have stored that info somewhere.

**Reply by tjahnn** [reply]
Thank you

**Comment by JaffaKetchup** [comment, marked as answer]
I don't think so. 'flutter_map' can't parse map information, due to the way static maps work.
You may have to find a 3rd-party API to do what you want.
Thanks for your interest, and we're happy to help in the future!

**Reply by tjahnn** [reply]
Thank you!
